src/nvim/tui/tui.c

```diff
@@ -935,11 +935,11 @@ static void tui_set_mode(UI *ui, ModeShape mode)
     }
   }
 
-  switch (shape) {
+  int shape;
+  switch (c.shape) {
```

Comment:
I kind of thought your PR would _only_ handle the EMSG thingy. Then I wouldn't have to change my PR which solely is about the cast warnings. :)

Reply:
I didn't see you changed your PR. Github seems a bit stochastic about sending the  "at-user pushed 1 commit" messages...

Reply:
Yeah. Anyway, please remove that part from your PR. Then each of our PRs does one logical change, which is a good thing, I guess.